cmd/kube-proxy/app/server.go

```diff
@@ -115,7 +117,8 @@ func NewProxyConfig() *ProxyServerConfig {
 		HealthzBindAddress: net.ParseIP("127.0.0.1"),
 		OOMScoreAdj:        qos.KubeProxyOOMScoreAdj,
 		ResourceContainer:  "/kube-proxy",
-		SyncPeriod:         30 * time.Second,
+		IptablesSyncPeriod: 30 * time.Second,
+		ConfigResyncPeriod: 10 * time.Minute,
```

Comment:
We decided to bump resyn period to 12 hours in production in controllers and that works well.
But maybe it needs to be lower in KubeProxy for some reason. If not, I think we can make it much higher.